domains/whsprr.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "whsprr-png",
+        "email": "musicbotqn@gmail.com"
+    },
+    "record": {
+        "CNAME": "whsprr-png.github.io"
```

Comment:
page gives 404 error, but if you do have your website hosted with github then comment with the website link

Reply:
whsprr-png.github.io/bio